bin/configure.sh

```diff
@@ -27,3 +27,82 @@ export NODE_PATH=../..:../../zlux-server-framework/node_modules:$NODE_PATH
 __UNTAGGED_READ_MODE=V6 $NODE_BIN initInstance.js
 
 APP_WORKSPACE_DIR=${INSTANCE_DIR}/workspace/app-server
+
+# Add static definition for zss. TODO: Needs routedServices + apiInfo
+cat <<EOF >${STATIC_DEF_CONFIG_DIR}/zss.ebcidic.yml
+#
+services:
+  - serviceId: zss
+    title: Zowe System Services Server (ZSS)
+    description: Zowe System Services Server (ZSS) is used for enabling low-level microservices and other privileged services
+    catalogUiTileId: zss
+    instanceBaseUrls:
+      - https://${ZOWE_EXPLORER_HOST}:${ZOWE_ZSS_SERVER_PORT}/
+    homePageRelativeUrl:
+    routedServices:
+      - gatewayUrl: /server/agent/environment # Displays ZSS host environment info
```

Comment:
All `gatewayUrl` should be with a prefix e.g. `/zss/api/v1/server/agent/environment`

Reply:
It doesnt seem that you need to prefix the name. Here is what I see on others
```
> cat datasets_ui.yml
#
services:
  - serviceId: explorer-mvs
    title: IBM z/OS Datasets UI
    description: IBM z/OS Datasets UI service
    catalogUiTileId:
    instanceBaseUrls:
      - https://hardcodedurl/
    homePageRelativeUrl:
    routedServices:
      - gatewayUrl: ui/v1
        serviceRelativeUrl: /ui/v1/explorer-mvs

#
services:
  - serviceId: explorer-jes
    title: IBM z/OS Jobs UI
    description: IBM z/OS Jobs UI service
    catalogUiTileId:
    instanceBaseUrls:
      - https://hardcodedurl/
    homePageRelativeUrl:
    routedServices:
      - gatewayUrl: ui/v1
        serviceRelativeUrl: /ui/v1/explorer-jes
```